Bryn Mawr Classical Review.js

```diff
@@ -3,93 +3,135 @@
 	"label": "Bryn Mawr Classical Review",
 	"creator": "Michael Berkowitz",
 	"target": "^https?://bmcr\\.brynmawr\\.edu/",
-	"minVersion": "1.0.0b4.r5",
+	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2015-06-02 10:52:41"
+	"lastUpdated": "2016-08-22 13:37:21"
 }
 
+/*
+	***** BEGIN LICENSE BLOCK *****
+	Copyright © 2016 Michael Berkowitz and John Muccigrosso
+	This file is part of Zotero.
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	***** END LICENSE BLOCK *****
+*/
+
 function detectWeb(doc, url) {
-	if (url.match(/by_reviewer/) || url.match(/by_author/) || url.match(/recent.html/) || url.match(/\/\d{4}\/$/)) {
+	if (url.search(/by_reviewer|by_author|recent\.html|\/\d{4}\/(index\.html)?$/) != -1) {
 		return "multiple";
-	} else if (url.match(/[\d\-]+\.html$/)) {
+	} else if (url.match(/[\d\-]+\.html$/) && ZU.xpathText(doc, '//h3/i')) {
 		return "journalArticle";
 	}
 }
 
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//*[@id="indexcontent"]//li//a');
+	Z.debug(rows.length);
```

Comment:
comment out (or if you think it's useful, add a descriptor, but I don't think so)